apps/backend/src/libs/modules/server-application/base-server-application.ts

```diff
@@ -170,6 +172,23 @@ class BaseServerApplication implements ServerApplication {
 					return reply.status(error.status).send(response);
 				}
 
+				if (isFastifyError(error)) {
```

Comment:
If you decide to leave this helper, than it should be named checkIsFastifyError
```suggestion
				if (checkIsFastifyError(error)) {
```